### DISCUSSION TITLE
Get build version as a variable

### DISCUSSION THREAD

**Discussion by Kptainflintt** [semaphoreui/semaphore#2172] [Q&A] [answered]
Hi,

Is it possible to get build version as a variable in playbook?

I use semaphore for Wordpress templates and I want to make "versions" of wordpress folder in order to revert if it's needed

So if I can use build version, I will use it to make version archives "tagged" with version build.

Thanks and very good job, semaphore is wondeful!

**Comment by jvincze84** [comment, marked as answer]
Hi,

I don't know what do you exaclty want, but this might help you.

Create a debug task:
```
    - name: Debug
      debug:
        msg: "{{semaphore_vars }}"
```
And see what happens:
```
TASK [Debug] *******************************************************************
ok: [10.8.0.6] => {
"msg": {
"task_details": {
"target_version": "1.0.2",
"type": "build",
"username": "admin"
}
}
}
```

I think you should use `{{ semaphore_vars.task_details.target_version }}`.

Br,
JVincze
